### DISCUSSION TITLE
Option to Open Item Detail as Modal

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3077] [Feature Requests]
When clicking on an item from Item Browse, it would be nice to have an option to open that Item Detail in a modal, so you don't lose track of where you are. Not sure if we always offer both options, or if it's a user/global/collection setting.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
